kinsumer.go

```diff
@@ -250,6 +259,21 @@ func (k *Kinsumer) dynamoCreateTableIfNotExists(name, distKey string) error {
 	if k.dynamoTableExists(name) {
 		return nil
 	}
+
+	if name == k.metadataTableName {
+		// If the metadata table is being created for the first time
+		// the loadShardIDsFromDynamo() call in refreshShards() will
+		// always return an empty shard list.
```

Comment:
Alternate idea: Would it not be better to call `loadShardIDsFromKinesis()` if `loadShardIDsFromDynamo()` returns an empty list?

Reply:
Yeah, that might be better. I made this change before replacing `DescribeStream` and I wanted to avoid accidentally hitting the rate limit but it should be fine with `ListShards`. I'll update the PR.

Reply:
Done.